CodeSnippetsPipeline.Test/CodeSnippetsPipeline.cs

```diff
@@ -3,24 +3,62 @@
 [TestFixture]
 public class CodeSnippetsPipeline
 {
+    // Assumption made below: snippetCategories = new string[] { "Stable", "KnownIssues"};
+
+    public static string [] GetStableTestsList(string language, string version)
```

Comment:
```suggestion
    private static HashSet<string> GetStableTestsList(string language, string version)
```
Since the collection is used for lookups, it may be better to return a HashSet with has an O(1) complexity.